From 8321ace9c2060b794364a000f12721e61fdb8832 Mon Sep 17 00:00:00 2001
From: Victor Gu <xigu@marvell.com>
Date: Tue, 29 Aug 2017 11:56:00 +0800
Subject: [PATCH 1205/1345] fix: mvebu: icu: fix the maximum number of ICU
 interrupts

commit  d12fd8ec185fdbbfe91e8116f4689de67d039630 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Fix the maximum number of ICU interrupts according to the function
specification - 208.

Change-Id: Ife62cb76963010e4c5fe1588572beb8daa2b6516
Signed-off-by: Victor Gu <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/43479
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/irqchip/irq-mvebu-icu.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/irqchip/irq-mvebu-icu.c b/drivers/irqchip/irq-mvebu-icu.c
index 88eca01..8d01d43 100644
--- a/drivers/irqchip/irq-mvebu-icu.c
+++ b/drivers/irqchip/irq-mvebu-icu.c
@@ -45,7 +45,7 @@
 
 #include <dt-bindings/interrupt-controller/mvebu-icu.h>
 
-#define ICU_MAX_IRQS		207
+#define ICU_MAX_IRQS		208
 #define ICU_MAX_SPI_IRQ_IN_GIC	128
 #define ICU_GIC_SPI_BASE0	64
 #define ICU_GIC_SPI_BASE1	288
-- 
1.7.9.5

